src/ray/common/test_util.cc

```diff
@@ -51,7 +51,7 @@ int TestSetupUtil::StartUpRedisServer(const int &port) {
     cmdargs.insert(cmdargs.end(), {"--loadmodule", TEST_REDIS_MODULE_LIBRARY_PATH});
   }
   cmdargs.insert(cmdargs.end(), {"--port", std::to_string(actual_port)});
-  RAY_LOG(INFO) << "Start redis command is: " << CreateCommandLine(cmdargs);
+  RAY_LOG(DEBUG) << "Start redis command is: " << CreateCommandLine(cmdargs);
```

Comment:
I'd probably leave these as `INFO` as they're only used for tests

Reply:
Makes it easier to spot-check failures in CI if they're a bit more verbose

Reply:
Process startup has historically been really noisy for us on the CI so I was hoping to cut down on that. Any thoughts on putting the message after the following `RAY_CHECK` (like `RAY_CHECK(...) << CreateCommandLine(cmdargs)`)?